test/unit/instance/changed.test.js

```diff
@@ -47,6 +47,21 @@ describe(Support.getTestDialectTeaser('Instance'), () => {
       expect(user.changed('meta')).to.equal(false);
     });
 
+    it('should return falsy for unchanged primitive via setDataValue', function() {
```

Comment:
move this one new test under `describe('setDataValue')`, as its testing `setDataValue`